user-registration-app/src/app/views/travel-reimbursement-view/travel-reimbursement-view.component.ts

```diff
@@ -1,13 +1,10 @@
 import { Component, OnInit } from '@angular/core';
-import { AddressValidator } from 'address-validator';
 import { HttpService } from '../../services/HttpService/HttpService';
```

Comment:
This file is our attempt at doing travel reimbursement right? this never worked and ended up not even being wanted at the end I think cause it's just easier for the finance director to do it on their own during the event. With that being said this can probably be deleted @rsquared226 check with the finance director and see if these is useful/wanted and go from there

Reply:
Yeah just checked and he said he'd rather keep it

Reply:
If someone can get the specs for what is needed in a travel reimbursement, we can try actually implementing it

Reply:
Yeah, we can def do this the next time we have an in-person event